src/server/utils/asynctaskexecutor.h

```diff
@@ -27,8 +23,8 @@ class AsyncTaskExecutor {
     static AsyncTaskExecutor* inst;
 };
 
-#define ASYNC_TASK(task) \
-    utils::AsyncTaskExecutor::instance()->submit_task(task);
+/*#define ASYNC_TASK(task) \
+    utils::AsyncTaskExecutor::instance()->submit_task(task);*/
 // TODO find something better
```

Comment:
Did you find something better?

Reply:
No way. I think that our work around is the only solution for our knowledge.

Reply:
Okok. Clean up the TODO then :man_shrugging: